multiqc/utils/config_defaults.yaml

```diff
@@ -256,7 +256,26 @@ fn_clean_exts:
   - "_NanoStats"
   - ".cutadapt"
   - ".mosdepth"
-  - _gopeaks
+  - "_gopeaks"
+  - "_metrics"
+  - ".metrics"
+  - ".vc"
+  - ".ploidy_estimation"
+  - ".wgs_contig_mean_cov"
+  - ".overall_mean_cov"
+  - "_coverage"
+  - ".wgs_coverage"
+  - ".wgs_fine_hist"
+  - ".fragment_length_hist"
+  - ".mapping"
+  - ".gc"
+  - ".trimmer"
+  - ".time"
```

Comment:
This one feels maybe a little risky - would it work if we moved it to `fn_clean_trim` so that it only gets removed if it's at the very end of a sample name? Just in case people have sample names like `mysample.timepoint1` for example 🤔

Reply:
I _think_ this should work as the suffixes are removed in order. So from `*.time_metrics.csv` the `_metrics.csv` would be removed from the `_metrics` above here, leaving `.time` to be removed afterwards by `fn_clean_trim`...

Reply:
Alternatively we could be less conservative and list the combinations here. eg. `vc_metrics`, `gc_metrics`, `time_metrics` etc..

Reply:
Okay, now it looks better

Reply:
The problem with `quant_metrics` is that we also have to add `quant.metrics.csv` due to the older DRAGEN format, and it will get a bit bloated. But maybe why not

Reply:
Great 👍🏻